packages/cli/src/lib/cmds/play.ts

```diff
@@ -145,11 +145,9 @@ class Play {
   */
 function validServiceAccountJsonFile(path: string | undefined): boolean {
   if (path == undefined) {
-    // TODO(@nohe427): Log an error
```

Comment:
**FYI: ** Error handling for this is handled on line 168 (of this file). I can throw errors here if needed.

Reply:
I'm happy with returning false if `path` is `null` or `undefined.

Reply:
Ahh, awesome. Thanks!